configure.ac

```
@@ -1709,6 +1709,17 @@ PHP_ADD_SOURCES(main, main.c snprintf.c spprintf.c \
        network.c php_open_temporary_file.c php_odbc_utils.c safe_bcmp.c \
        output.c getopt.c php_syslog.c, -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1)
 
+if printf "#if __ELF__\nelf\n#endif\n" | $CC -E - | grep elf > /dev/null; then
+  PHP_ADD_SOURCES(main, gdb_inlined_script.c)
+
+  cat >> Makefile.objects <<EOF
+$abs_srcdir/main/gdb_inlined_script.c: $abs_srcdir/scripts/gdb/gen_gdb_inlined_script.php $abs_srcdir/scripts/gdb/php_gdb.py $abs_srcdir/scripts/gdb/php_gdb.gdb
+	@if test ! -z "\$(PHP)"; then \\
+		\$(PHP) $abs_srcdir/scripts/gdb/gen_gdb_inlined_script.php; \\
+	fi;
+EOF
+fi
+
```

This is perhaps a bit better check, if I'll be able to sort out those wrong CFLAGS and LIBTOOL flags (the `$standard_libtool_flag` variable). So, it's just a possible suggestion for the future, because it doesn't yet work.

```suggestion
AC_CHECK_DECL([__ELF__], [
  PHP_ADD_SOURCES(main, gdb_inlined_script.c)

  cat >> Makefile.objects <<EOF
$abs_srcdir/main/gdb_inlined_script.c: $abs_srcdir/scripts/gdb/gen_gdb_inlined_script.php $abs_srcdir/scripts/gdb/php_gdb.py $abs_srcdir/scripts/gdb/php_gdb.gdb
	@if test ! -z "\$(PHP)"; then \\
		\$(PHP) $abs_srcdir/scripts/gdb/gen_gdb_inlined_script.php; \\
	fi;
EOF
])
```